### PR TITLE
Delete APPENGINE.md

### PR DESCRIPTION
**Pull request by lesv**
No longer required

### CODE REVIEW

**Comment by garrettjonesgoogle** [comment]
What made it not required?

**Comment by lesv** [comment]
#2419 Fix GAE instructions

**Comment by garrettjonesgoogle** [comment]
#2419  doesn't seem to indicate that the instructions should be completely removed - it seems like there is some nuance still...

**Comment by lesv** [comment]
@garrettjonesgoogle The other PR you suggested I should remove them completely (and fix the master README not to point at them).

**Comment by lesv** [comment]
See #2456

**Comment by garrettjonesgoogle** [comment]
Ok, https://github.com/GoogleCloudPlatform/google-cloud-java/pull/2456 makes more sense for context.

**Comment by garrettjonesgoogle** [comment]
So pulling the pieces together: We can essentially delete this because Java 7 support on App Engine is deprecated, so nobody should be setting this up as a new project any more, right?

File: README.md [hunk 1]
```diff
@@ -72,7 +72,7 @@ libraryDependencies += "com.google.cloud" % "google-cloud" % "0.33.0-alpha"
 It also works just as well to declare a dependency only on the specific clients that you need. See the README of
 each client for instructions.
 
-For running on Google App Engine, see [more instructions here](./APPENGINE.md).
+These client librareis can be used on App Engine standard Java 8 runtime, App Engine flexible (including the Compat runtime), but not on the App Engine standard Java 7 runtime.
```
Comment (garrettjonesgoogle) [inline README.md]: spelling (librareis)

Also, some of the libraries can technically be used on GAE Java 7 (e.g. datastore, storage, bigquery).

**Comment by lesv** [comment]
PTAL @garrettjonesgoogle

**Comment by lesv** [comment]
Yes to - delete this, right?

**Comment by hzyi-google** [comment]
Fix #2419.

**Pull request merged**